.travis.yml

```diff
@@ -67,7 +64,7 @@ jobs:
             --name fencer-server
             --network test
             --detach
-            juspay/fencer
+            juspayin/fencer
```

Comment:
Even though it is not a big deal, I'd suggest using an environment variable instead of a string literal.